.coveragerc

```diff
@@ -238,6 +238,7 @@ omit =
     homeassistant/components/notify/xmpp.py
     homeassistant/components/nuimo_controller.py
     homeassistant/components/openalpr.py
+    homeassistant/components/remote/harmony.py
```

Comment:
same as the one above, I'm hoping the 644 was a typo and it should be 664, thats what a majority of the other components files seemed to be